pkg/pep440/range.go

```diff
@@ -6,39 +6,43 @@ import (
 	"unicode"
 )
 
+//go:generate stringer -linecomment -type op
```

Comment:
https://github.com/quay/claircore/blob/b10fa7900054a953584c68777bff9c919406af00/pkg/pep440/range.go#L58 could maybe simplify this now that it's been stringered.